### ISSUE TITLE
Add flag to allow queries on slave instances.

### ISSUE THREAD

**Issue by ibarsi** [variety/variety-cli#10]
Currently, there's no easy way (as far as I know) to query against slave instances in a replica set. If you try, you get the following error message:

```
E QUERY    [thread1] Error: listCollections failed: {
	"ok" : 0,
	"errmsg" : "not master and slaveOk=false",
	"code" : 13435,
	"codeName" : "NotMasterNoSlaveOk"
}
```

I think there should be a flag that allows you to enable slave queries. That way, this tool can easily analyze replica sets when you'd prefer not to connect to the primary.

**Comment by todvora** [comment]
Perfect, thank you for your contribution @ibarsi!

Released as 0.4.2 and available in npm registry.

**Comment by JamesCropcho** [comment]
Lovely. On behalf of all the Variety afficianados, thank you very much, Igor ( @ibarsi ).
